libraries/Microsoft.Bot.Builder/EventFactory.cs

```diff
@@ -80,7 +80,7 @@ private static Activity CreateHandoffEvent(string name, object value, Conversati
             var handoffEvent = Activity.CreateEventActivity() as Activity;
 
             handoffEvent.Name = name;
-            handoffEvent.Value = JObject.FromObject(value);
```

Comment:
It would probably be just fine. It also makes more sense to do it this way, since you don't make any assumptions about skills that shouldn't be made. I'll make the change.